### DISCUSSION TITLE
Başlıkların Belirlemesi Konusunda Tartışma / TR

### DISCUSSION THREAD

**Discussion by damla** [sdtrdev/source-pocket#33] [Ideas]
Selamlar herkese 👋🏼,

Bildiğiniz üzere bu projenin temel amacı, yazılım ve donanım ile ilgilenen geliştiriciler için bilgilendirici, yararlı ve kullanışlı kaynakların bulunduğu linklerin paylaşımını açık kaynak bir şekilde sağlamak. Burada ise tartışmak istediğim konu, **Source Pocket**' ın başlıklarının ne şekilde düzenlenmesi gerektiği. 🧐

Şu ana kadar aklımda [Software Development Turkey](https://discord.gg/J3PTmeFj6s) sunucumuzda bulunan başlıkları taşımak vardı fakat bu sefer aynı anda birden fazla alana hitap eden kaynaklar konusunda sıkıntılar oluşabiliyor ve kaynak tekrarına sebebiyet verebiliyor. Örnek olarak, [w3schools](https://www.w3schools.com) birden çok dile hitap eden bir kaynak ve bunu her dil için tekrar tekrar belirtmek **gereksiz kalabalık** ve **yer maliyeti** demektir. 💰

Sonrasında meslek gruplarına göre ayırmanın da mantıklı olabileceğini düşündüm. Örnek olarak, **Front-End Web Development**, **Back-End Web Development**, **DevOps** vs. gibi bölünebilir. Fakat bu da sektöre yeni giriş yapmış arkadaşlar için başlangıçta kompleks gelebilir. 😬

Son olarak "Neden ikisi de olmasın?" diyerekten, [Software Development Turkey](https://discord.gg/J3PTmeFj6s) bulunan başlıkları yine eklemeyi fakat oralardaki konulara **sadece o başlıkla ilgili spesifik kaynakların paylaşılmasının**, meslek alanındaki başlıklara ise zaten genel bir konu olmasından kaynaklı olarak **daha genel kaynak paylaşımının yapılmasının** uygun olabileceğini düşündüm. 🤔

Ben düşündüm fakat sizlerin fikrini de merak ettiğim için bu konudaki en uygun çözümü bulabilmek adına bu tartışmayı başlattım. 🏁

Bu konuda katkı sağlayabilmek adına, buradaki [linkten](https://linkto.run/p/ZO92DQDP) ankete katılıp, mevcut fikirler üzerinden oylama yapabilirsiniz ya da "Benim başka bir fikrim var 🧐" diyorsanız tartışmaya katılabilirsiniz 🙃

**Comment by omerayyildiz** [comment]
Sık sık güncelleyebilirsek çok işe yarar olacaktır.

**Comment by ilter** [comment]
LGTM, yeni başlayan arkadaşlar için bi road map görevi olabilir.
